src/scripts.js

```diff
@@ -1,24 +1,41 @@
 import './css/base.scss';
 import './css/styles.scss';
 
-import recipeData from './data/recipes';
-import ingredientData from './data/ingredients';
-import users from './data/users';
-
 import Pantry from './pantry';
 import Recipe from './recipe';
 import User from './user';
 import Cookbook from './cookbook';
 
+let users = fetch("https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData")
+  .then(response => response.json())
+  .then(data => data.wcUsersData)
+  .catch(error => error.message)
```

Comment:
Will this print the error message somewhere or do we need to console log it? Do we want all of our error messages to be error.message or more like the strings you have below? Might be worthwhile to make them all the same structure.

Reply:
I will need to add a console.log to the error message to display the error if an error occurs